.gitignore

```diff
@@ -62,4 +62,13 @@ localnet_config.yaml
 # Relase artifacts produced by `ignite chain build --release`
 release
 
+# SMT KVStore files
+smt
+
+# Do not allow a multi-moduled projected
 go.work.sum
+
+# TODO_IN_THIS_COMMIT: Why did we start generating .dot files?
```

Comment:
@Olshansk I don't see any `.dot` files in my working tree. Is it possible that something other than running the E2E relay locally produced them on your workstation?

Reply:
I started getting them at one point o-o

Reply:
@h5law 
> Can you share the output of find . -type f -iname '*.dot'?

Reply:
For me it was these two: `rm pkg/client/tx/debug_container.dot pkg/client/supplier/debug_container.dot`

However, I can't repro it now... Ran unit tests, e2e test and localnet.